bw2io/export/gexf.py

```diff
@@ -109,13 +185,30 @@ def get_data(self, E):
 
 
 class DatabaseSelectionToGEXF(DatabaseToGEXF):
-    """Export a Gephi graph for a selection of activities from a database.
+    # """Export a Gephi graph for a selection of activities from a database.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)